src/Localization/Localization/src/ResourceManagerStringLocalizer.cs

```diff
@@ -166,16 +166,32 @@ protected IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures,
 
         if (_missingManifestCache.ContainsKey(cacheKey))
         {
+            Log.ResourceNotFound(_logger, name);
+
             return null;
         }
 
         try
         {
-            return _resourceManager.GetString(name, culture);
+            var value = _resourceManager.GetString(name, culture);
+
+            if (name == value)
+            {
+                Log.ResourceFound(_logger, name);
+            }
+            else
+            {
+                Log.ResourceNotFound(_logger, name);
+            }
+
+            return value;
         }
         catch (MissingManifestResourceException)
         {
             _missingManifestCache.TryAdd(cacheKey, null);
+
+            Log.ResourceNotFound(_logger, name);
```

Comment:
Should we provide a different log for a missing manifest vs a non-localized resource where the `name == value`?

@DamianEdwards do you have an opinion on this?

Reply:
No real opinion, I'd defer to folks who've been using it for real to inform us to how useful that would be.

Reply:
IMHO we should log for both

Reply:
I agree we should log in both cases. I was asking if there should be a different log message for the two different calls to `Log.ResourceNotFound` since they are called for slightly different reasons, but I'm fine with using the same message whenever a resource is not found for any reason like the PR does currently.

Reply:
Or we could change the log level for missing manifest to `trace` so in debug the missing resources will be logged which are useful in almost cases

Reply:
I'd just leave them exactly the same for now. If we were going to change the log level, we'd want to give it a unique event id and log message. Changing the level of a log but nothing else would be confusing.